_includes/footer.html

```diff
@@ -28,7 +28,7 @@
           <a href="https://www.idmanagement.gov/about-us/" class="usa-identifier__required-link usa-link">About IDM</a>
```

Comment:
@claytonjbarnette - GSA sets the footer titles and format. Please remove these updates.

Reply:
👍